common-test-lib/src/main/java/com/google/cloud/tools/intellij/testing/BasePluginTestCase.java

```diff
@@ -26,18 +26,17 @@
 import com.intellij.openapi.extensions.impl.ExtensionsAreaImpl;
 import com.intellij.openapi.project.Project;
 import com.intellij.testFramework.TestRunnerUtil;
-
 import org.jetbrains.annotations.NotNull;
 import org.junit.After;
 import org.junit.Before;
 import org.junit.runner.RunWith;
-import org.mockito.runners.MockitoJUnitRunner;
+import org.mockito.junit.MockitoJUnitRunner;
 import org.picocontainer.MutablePicoContainer;
 
 /**
  * Test base class that provides a mock Intellij application and project.
  */
-@RunWith(MockitoJUnitRunner.class)
+@RunWith(MockitoJUnitRunner.Silent.class)
```

Comment:
Ah yes, this is the one I was thinking of. I removed it and only 1 test failed (`DefaultCloudSdkVersionNotifierTest.testNotifyIfCloudSdkNotSupported_nullSdkPath`) due to an `UnnecessaryStubbingException`, so I fixed that test and removed the `Silent` runner. PTAL